tests/test_ote_api.py

```diff
@@ -222,7 +223,10 @@ def progress_callback(progress: float, score: Optional[float] = None):
         # stopping process has to happen in less than 35 seconds
         train_future.result()
         self.assertEqual(training_progress_curve[-1], 100)
-        self.assertLess(time.time() - start_time, 100, 'Expected to stop within 100 seconds.')
+        if os.getenv("CUDA_VISIBLE_DEVICES") == "":
+            self.assertLess(time.time() - start_time, 400, 'Expected to stop within 400 seconds.')
+        else:
+            self.assertLess(time.time() - start_time, 100, 'Expected to stop within 100 seconds.')
```

Comment:
2 @aiarkinx : I propose to do as follows
```python
if os.getenv("CUDA_VISIBLE_DEVICES") == "":
    timeout = 400
else:
    timeout = 100

self.assertLess(time.time() - start_time, timeout, f'Expected to stop within {timeout} seconds.')
```
(please, do not move complicated code inside `if - then - else` clause if it is possible to move only one variable into it.)

Reply:
done